substrate/frame/revive/mock-network/src/parachain/contracts_config.rs

```diff
@@ -15,13 +15,23 @@
 // along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
 
 use super::{Balances, Runtime, RuntimeCall, RuntimeEvent};
-use crate::parachain::RuntimeHoldReason;
+use crate::parachain::{RuntimeHoldReason,parameter_types};
 use frame_support::derive_impl;
+use pallet_revive::precompiles::builtin::xcm::XcmPrecompile;
+use xcm_builder::FixedWeightBounds;
+use crate::parachain::MaxInstructions; 
+use xcm::latest::Weight;
+
+parameter_types! {
+	pub const BaseXcmWeight: xcm::latest::Weight = Weight::from_parts(100_000_000, 2_000);
+}
 
 #[derive_impl(pallet_revive::config_preludes::TestDefaultConfig)]
 impl pallet_revive::Config for Runtime {
 	type AddressMapper = pallet_revive::AccountId32Mapper<Self>;
 	type Currency = Balances;
 	type Time = super::Timestamp;
 	type Xcm = pallet_xcm::Pallet<Self>;
-}
+	type Precompiles = (XcmPrecompile<Self>,);
+	type XcmWeigher = FixedWeightBounds<BaseXcmWeight, RuntimeCall, MaxInstructions>;
```

Comment:
not sure if should use `FixedWeightBounds` or [WeightInfoBounds](https://github.com/paritytech/polkadot-sdk/blob/c679ed79a1de575d4c3d5b8878b3e401b024f2f2/polkadot/xcm/xcm-builder/src/weight.rs#L76), but can easily be changed later, not blocking now

Reply:
This is testing code, `FixedWeightBounds` is fine here